app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
- [ ] Hi @ahadb802, let's kindly share a bit together concerning these associations. For example, if you ask for `User.first.posts` Rails will go looking in the posts table for a foreign_key called user_id (which does not exist in the posts table). To get it pointing to the right table, kindly specify foreign_key like below

`
has_many :posts, foreign_key: 'author_id'`